src/diffusers/loaders/transformer_sd3.py

```diff
@@ -0,0 +1,261 @@
+import inspect
+from functools import partial
+from typing import Dict, List, Optional, Union
+
+import torch.nn as nn
+
+from ..utils import (
+    USE_PEFT_BACKEND,
+    delete_adapter_layers,
+    is_accelerate_available,
+    logging,
+    set_adapter_layers,
+    set_weights_and_activate_adapters,
+)
+from .lora import TEXT_ENCODER_NAME, TRANSFORMER_NAME
+
+
+if is_accelerate_available():
+    from accelerate.hooks import AlignDevicesHook, CpuOffload, remove_hook_from_module
+
+logger = logging.get_logger(__name__)
+
+
+class SD3TransformerLoadersMixin:
```

Comment:
Nit. Could we name this to `SD3TransformerLoRALoadersMixin`. Conveys what it is responsible for loading a bit more explicitly